go.mod

```diff
@@ -8,7 +8,7 @@ replace github.com/vmware-tanzu/vm-operator/pkg/constants/testlabels => github.c
 
 require (
 	github.com/vmware-tanzu/net-operator-api v0.0.0-20240326163340-1f32d6bf7f9d
-	github.com/vmware-tanzu/nsx-operator/pkg/apis v0.1.0
+	github.com/vmware-tanzu/nsx-operator/pkg/apis v0.0.0-20240816055841-8081971d43ea
```

Comment:
While this is only an API package:
Question: should we pick a random commit for this? Will they have a released version soon?

Reply:
The other way would be to do it similar to how we do for vm-operators topologyv1 and have an internal copy.

cc @sbueringer @fabriziopandini .

Reply:
There will not be a tag in a short time and they only promise 3 CRDs we are using can have the final version this week. Internal copy does not work because they will change spec, status field names.

Reply:
If we are only using the API types from there I would prefer using a commit over copying the content of the commit.

The reason why we did this in vm-operator was because we are using vm-operator in a **lot** of other places as well

Reply:
Could you point me the example? I only see vm-operator tag in the go mod https://github.com/kubernetes-sigs/cluster-api-provider-vsphere/blob/75a5e72691d58f48027203b2c0afbc989390b8aa/go.mod#L13